catalog/spatial/kml/spatial-kml-transformer/src/test/java/org/codice/ddf/spatial/kml/converter/TestKmlDocumentToJtsGeometryConverter.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.kml.converter;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.io.InputStream;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import de.micromata.opengis.kml.v_2_2_0.Kml;
+import de.micromata.opengis.kml.v_2_2_0.Placemark;
+
+public class TestKmlDocumentToJtsGeometryConverter {
+    private static de.micromata.opengis.kml.v_2_2_0.Document testKmlDocument;
+
+    @BeforeClass
+    public static void setup() {
+        InputStream stream = TestKmlDocumentToJtsGeometryConverter.class.getResourceAsStream(
+                "/kmlDocument.kml");
+
+        de.micromata.opengis.kml.v_2_2_0.Kml kml = Kml.unmarshal(stream);
+        assertThat(kml, is(notNullValue()));
+
+        de.micromata.opengis.kml.v_2_2_0.Document document =
+                ((de.micromata.opengis.kml.v_2_2_0.Document) kml.getFeature());
+        assertThat(document, is(notNullValue()));
```

Comment:
Could be wrong, but I'm not sure we should have this assertion here in the beforeclass method

Reply:
I had them there because the stuff being asserted is needed for the tests.
But I guess it will fail anyway.
I can remove them.